frontend/components/bc/format/FormatPercent.vue

```diff
@@ -52,7 +52,7 @@ const data = computed(() => {
   }
   label = formatPercent(percent, config)
   if (props.comparePercent !== undefined) {
-    if (Math.abs(props.comparePercent - percent) <= 0.5) {
+    if (Math.abs(props.comparePercent - percent) <= 0.25) {
       className = 'text-equal'
       leadingIcon = faArrowsLeftRight
       compareResult = 'equal'
```

Comment:
this would be easier to understand, but I was a bit too chatty here I guess
```suggestion
    const thresholdToDifferenciateUnderperformerAndOverperformer = 0.25
    if (Math.abs(props.comparePercent - percent) <= thresholdToDifferenciateUnderperformerAndOverperformer) {
      className = 'text-equal'
      leadingIcon = faArrowsLeftRight
      compareResult = 'equal'
```